### DISCUSSION TITLE
GitHub OAuth application - is this safe?

### DISCUSSION THREAD

**Discussion by KristiNielson93** [community/community#65233] [New to GitHub]
A pop-up window keeps appearing from GitHub Oauth saying I can authorize it. Is this safe?

**Comment by lab596** [comment]
GitHub OAuth (Open Authorization) is a secure method for granting third-party applications limited access to your GitHub account without sharing your credentials. It's commonly used for integrations, allowing services to interact with your GitHub account on your behalf. If you encounter a pop-up window asking you to authorize an OAuth application, it's important to exercise caution and follow best practices to ensure your account's security.

Here are some steps and considerations:

1. Check Application Source: Make sure the application requesting authorization is from a reputable source. If it's from a known and trusted service, it's generally safe to proceed.

2. Verify Permissions: Before granting authorization, review the permissions the application is requesting. Only grant the permissions that are necessary for the app's functionality. If the requested permissions seem excessive or unnecessary for the app's purpose, it's a red flag.

3. Use GitHub's OAuth: If the pop-up is directly from GitHub and you're interacting with GitHub's OAuth, it's generally safe. However, always double-check the URL to ensure you're on the official GitHub domain (github.com).

**Reply by Buggem** [reply]
AI real
